AfricanReligions/events/afr_misc_events.txt

```diff
@@ -235,4 +235,219 @@ character_event = {
             }
         }
     }
+}
+
+# Bori Selecting Patron Deities events
+character_event = {
+    id = AFRREL.00500
+    desc = EVTDESC_AFRREL00500
+    border = GFX_event_normal_frame_religion
+    picture = GFX_evt_desert
+
+    is_triggered_only = yes
+
+    immediate = {
+        if = {
+            limit = { if_playable = yes }
+            any_courtier = {
+                limit = {
+                    prisoner = no
+                    religion = root
+                    age = 16
+                    has_minor_deity_trigger = no
+                }
+                character_event = { id = AFRREL.00500 days = 1 }
+            }
+        }
+    }
+
+    option = {
+        name = EVTOPTA_AFRREL00501
+        hidden_tooltip = { character_event = { id = AFRREL.00510 } }
+    }
+    option = {
+        name = EVTOPTA_AFRREL00502
+        piety = 25
+        remove_character_modifier = picked_main_deity_timer
```

Comment:
This line needs to be in a `hidden_tooltip` or else it will be displayed to the player.